packages/flutter/lib/src/material/switch.dart

```diff
@@ -612,7 +635,7 @@ class Switch extends StatelessWidget {
       value: value,
       onChanged: onChanged,
       size: _getSwitchSize(context),
-      activeColor: effectiveActiveThumbColor,
+      activeThumbColor: effectiveActiveThumbColor,
```

Comment:
```suggestion
      activeThumbColor: activeThumbColor ?? effectiveActiveThumbColor,
```

We do want to apply `activeThumbColor` to the Cupertino switch if it is set. The main difference between the two switches use `activeColor` if another property is null.

Reply:
Updated, thank you.